fastlane/lanes/build.rb

```diff
@@ -28,27 +70,15 @@
 
   next unless create_release
 
-  archive_zip_path = File.join(File.dirname(Dir.pwd), 'Simplenote.xarchive.zip')
-  zip(path: lane_context[SharedValues::XCODEBUILD_ARCHIVE], output_path: archive_zip_path)
+  xcarchive_zip_path = File.join(OUTPUT_DIRECTORY_PATH, 'Simplenote.xarchive.zip')
+  zip(path: XCARCHIVE_PATH, output_path: xcarchive_zip_path)
 
-  version = beta_release ? ios_get_build_version : ios_get_app_version(public_version_xcconfig_file: VERSION_FILE_PATH)
+  version = beta_release ? build_code_current : release_version_current
   create_release(
```

Comment:
I can see a further iteration where uploads to TestFlight and Sentry run in parallel, and we only create the release on GitHub when the TestFlight step has succeeded.